src/components/LeftSidebar/left-sidebar.css

```diff
@@ -12,9 +19,9 @@
 }
 
 .sidebar-list-item:hover{
```

Comment:
On hovering over this, based on the screen size the hover effect works differently,
attaching a screenshot for your reference
![aside_abhishek](https://user-images.githubusercontent.com/95222845/168817909-92aea418-c1eb-4ad0-bf6c-6aac53ad732b.jpg)

Reply:
fixed